config/keys

```diff
@@ -11,6 +11,10 @@ nmap f follow
 nmap F win-follow
 nmap gg scroll-top
 nmap gi focus-input
+nmap gu go-parent-dir
+nmap gU go-root-dir
+nmap g- url-decrement
```

Comment:
These shorcuts are `<C-x>` and `<C-a>` in vimperator.
Are they different in Chromium vim?